ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/433__data-science-pipelines-general.robot

```diff
@@ -0,0 +1,80 @@
+*** Settings ***
```

Comment:
I think you could remove "general" from the robot file name, to remain consistent with the other files (although we don't have a such consistency over all the files in ods-ci, but it should be good to start having)